lib/game.rb

```diff
@@ -139,20 +151,33 @@ def player_has_lost?
   def cpu_has_lost?
     @cpu_cruiser.sunk? && @cpu_submarine.sunk?
   end
+
+  def clear
```

Comment:
I'm glad you researched how to do this on google in order to make our UI more friendly.